esp/services/ws_sql/SQL2ECL/HPCCFileCache.cpp

```diff
@@ -23,6 +23,26 @@ HPCCFileCache * HPCCFileCache::createFileCache(const char * username, const char
     ESPLOG(LogMax, "WsSQL: Creating new HPCC FILE CACHE");
     return new HPCCFileCache(username,passwd);
 }
+void populateColums(IArrayOf<HPCCColumnMetaData> * cols, IArrayOf<IEspHPCCColumn> & pColumns)
+{
+    for (int i = 0; i < cols->length(); i++)
+    {
+       Owned<IEspHPCCColumn> pCol = createHPCCColumn();
+       HPCCColumnMetaData& currcol = cols->item(i);
+
+       IArrayOf<HPCCColumnMetaData> * ccols = currcol.getChildColumns();
```

Comment:
Any chance of child columns being null here?

Reply:
avoided.